.vortex/installer/tests/Unit/Utils/FileManagerTest.php

```
@@ -209,17 +209,17 @@ public function testPrepareDemoNotDemoMode(): void {
     $this->assertEquals('Not a demo mode.', $result);
   }
 
-  public function testPrepareDemoWithDownloadSkip(): void {
+  public function testPrepareDemoWithFetchSkip(): void {
     $config = new Config('/tmp/root', self::$sut, '/tmp/tmp');
     $config->set(Config::IS_DEMO, TRUE);
-    $config->set(Config::IS_DEMO_DB_DOWNLOAD_SKIP, TRUE);
+    $config->set(Config::IS_DEMO_DB_FETCH_SKIP, TRUE);
     $fm = new FileManager($config);
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $fm. Configured minimum length is 3. (undefined)

(ShortVariable)